docs/source/API/core/view/view.rst

```diff
@@ -498,6 +535,29 @@ Other
    With the unmanaged view, there is no guarantee that referenced
    address is valid, only that it is a non-null pointer.
 
+MDSpan Conversions
+~~~~~~~~~~~~~~~~~~
+
+.. cppkokkos:function:: template <class OtherElementType, class OtherExtents, class OtherLayoutPolicy, class OtherAccessor> constexpr operator mdspan<OtherElementType, OtherExtents, OtherLayoutPolicy, OtherAccessor>()
+
+   :tparam OtherElementType: the target MDSpan element type
+   :tparam OtherExtents: the target MDSpan extents
+   :tparam OtherLayoutPolicy: the target MDSpan layout
+   :tparam OtherAccessor: the target MDSpan accessor
+
+   :constraints: :cpp:class:`View`'s `natural mdspan <#Natural MDSpans>`_ must be assignable to :cpp:`mdspan<OtherElementType, OtherExtents, OtherLayoutPolicy, OtherAccessor>`
+
+   :returns: an MDSpan with extents and a layout converted from the :cpp:class:`View`'s *natural mdspan*.
+
+.. cppkokkos:function:: template <class OtherAccessorType = Kokkos::default_accessor<typename traits::value_type>> constexpr auto to_mdspan(const OtherAccessorType& other_accessor = OtherAccessorType{})
+
+   :tparam OtherAccessor: the target MDSpan accessor
+
+   :constraints: :cpp:`typename OtherAccessorType::data_handle_type` must be assignable to :cpp:`value_type*`
```

Comment:
TIL about sphinx is that we can put whatever heading here (like "constraints", "preconditions", etc) and it will be styled the same as the template parameters and returns